Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -214,6 +235,23 @@ open class InputTextView: UITextView, AppearanceProvider {
         isScrollEnabled = true
     }
 
+    // MARK: - Link Detection
+
+    /// Resets the text attributes whenever the text changes.
+    open func resetAttributes() {
```

Comment:
Why do we need to do this?

Reply:
We are changing the color of the text, whenever we have a link in the input. When thera are new changes, the previous range of the text, will have the same blue color, if we do not reset it. I'll make sure I add a comment explaining why we need this 👍

Reply:
@laevandus Good point, the function is named `resetAttributes` because initially I was actually resetting all `textStorage` attributes, but that did not work out very well, so I changed it 😄 Either way, I'm going to test a different approach which I'm testing out and I'll as for a re-review then 👍 Will keep in mind your suggestions as wel 👍

Reply:
This has now been simplified 👍